### ISSUE TITLE
line-dasharray problem - Mapbox GL export

### ISSUE THREAD

**Issue by citizenfish** [tomchadwin/qgis2web#855]
This is using the OS Mastermap light style in QGIS and exporting the topographicline layer https://github.com/OrdnanceSurvey/OSMM-Topography-Layer-stylesheets/tree/master/Schema%20version%209/Stylesheets/QGIS%20stylesheets%20(QML)

```
An error has occurred while executing Python code: 

KeyError: 'line-dasharray' 
 .... 
  File "qgis2web/mapboxStyleScripts.py", line 77, in getLayerStyle

    paintProps[prop].append(layer["filter"])

    KeyError: 'line-dasharray'

Python version: 3.7.3 (default, Mar 27 2019, 09:23:15) [Clang 10.0.1 (clang-1001.0.46.3)] 
QGIS version: 3.10.0-A Coruña A Coruña, 6c816b4204 

```

**Comment by tomchadwin** [comment]
OK, I'm coding a tad blind here, as I don't have Mastermap to hand to test. Can you give it a go? Two things to check:

1. Is the error gone?
2. Is the style element which caused the error (line style) actually implemented in the exported map?

**Comment by citizenfish** [comment]
Can confirm that the python error is now gone

Next problem is that the style element is NOT implemented. So I see  a console error of:-

![Screenshot 2020-03-09 at 16 03 20](https://user-images.githubusercontent.com/10531204/76233064-87cba580-621f-11ea-8743-a4806c1271bc.png)

**Comment by tomchadwin** [comment]
Released as 3.13.0 and pushed to QGIS plugins repo.
